packages/ui-brand/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@hackoregon/ui-brand",
+  "sideEffects": false,
+  "version": "0.0.4",
+  "description": "Brand assets for CIVIC",
+  "main": "dist/index.js",
+  "modules": "es/index.js",
+  "scripts": {
+    "build:esm": "rimraf es && babel src --out-dir es --copy-files --no-comments",
+    "build:cjs": "rimraf dist && babel src --out-dir dist --copy-files --no-comments",
+    "build": "BABEL_ENV=esm yarn run build:esm && BABEL_ENV=cjs yarn run build:cjs",
+    "configure": "yarn run build",
```

Comment:
I suspect this is copied from another package so I won't scrutinize but I think some of these scripts can be rethought and maybe trimmed back.

Reply:
Will be addressed in #1208